apps/music/index.html

```diff
@@ -156,7 +156,7 @@ <h1 id="title-text" data-l10n-id="music"></h1>
           <span data-l10n-id="search-titles">TrackS</span>
             (<span class="search-result-count"></span>)
         </gaia-subheader>
-        <div class="search-results"></div>
+        <div class="search-results" role="listbox"></div>
```

Comment:
@eeejay There are two more elements use `search-results` class and we should also add `role="listbox"` to them?